Robitaille/template/simple_functions.py

```diff
@@ -10,3 +10,7 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+
+def IsPrime(value, option=None):
```

Comment:
This is good to add, but it does not follow PEP8 Python style considerations.  (there are different style guides for Python versus Java, maybe you're more familiar with Java? Consider looking at [PEP8](https://www.python.org/dev/peps/pep-0008/).

At any rate, this is more consistent with our style guidelines:
```suggestion
def is_prime(value, option=None):
```